_gsdocs-proposals/2019/proposal_Rucio.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Rucio - Exascale data management
+layout: gsdocs_proposal
+project: Rucio
+year: 2019
+organization:
+ - UOslo
+ - CERN
+ - RAL
+---
+
+## Description
+
+Rucio is a software framework that provides functionality to organize, manage, and access large volumes of scientific data using customisable policies. The data can be spread across globally distributed locations and across heterogeneous data centers, uniting different storage and network technologies as a single federated entity. Rucio offers advanced features such as distributed data recovery or adaptive replication, and is highly scalable, modular, and extensible. Rucio has been originally developed to meet the requirements of the high-energy physics experiment ATLAS, and is continuously extended to support LHC experiments and other diverse scientific communities. For example, the ATLAS experiment orchestrated an Exabyte of data transfer and processing and is growing rapidly.
+
+The current documentation is available from multiple places and in different formats, including readthedocs.io, Google Docs, Github, DockerHub, and Wikis. This dispersion and diversity makes it difficult to pinpoint information and to recognise which information is outdated or superseded, wrong, or simply lacking in detail. With this proposal we aim to automate this process as much as possible, as we think that the best documentation is the one which can be generated from the code itself. In addition, the manually written documentation, e.g., setup and tutorials, should also be brought into a single format from which it can be distributed automatically. We target the following structure with single entry points:
```

Comment:
I like this part. It really explain the current problem and the following part which explains the target structure. However, it is not clear to me if you expect the candidate to put in place the automation part. Also, I miss a `Task` section with concrete tasks or ideas to be done during the program. This could also provide a clearer picture of the project to the candidates so they can also propose ideas or refine the existing ones. 

You could have a look at this [link](https://developers.google.com/season-of-docs/docs/project-ideas), where Google suggests some content for the proposal ideas.

Reply:
added